LayoutTests/TestExpectations

```diff
@@ -5984,3 +5984,12 @@ webkit.org/b/243562 imported/w3c/web-platform-tests/css/css-ui/compute-kind-widg
 imported/w3c/web-platform-tests/html/semantics/scripting-1/the-script-element/css-module/charset.html [ Pass Failure ]
 imported/w3c/web-platform-tests/html/semantics/scripting-1/the-script-element/css-module/cors-crossorigin-requests.html [ Pass Failure ]
 imported/w3c/web-platform-tests/html/semantics/scripting-1/the-script-element/json-module/cors-crossorigin-requests.html [ Pass Failure ]
+
+# These are timeouts after adding wheel actions to WebKitTestRunner: webkit.org/b/243272
```

Comment:
Describing this as adding wheel actions "to WebKitTestRunner" is misleading. It would be more appropriate to say "in WPT vendor JS".

Reply:
Fixed.